composer/models/resnet/resnet_hparams.py

```diff
@@ -32,7 +32,6 @@ class ResNetHparams(ModelHparams):
     model_name: str = hp.optional(
         f"ResNet architecture to instantiate, must be one of {ComposerResNet.valid_model_names}. (default: '')",
         default='')
-    num_classes: int = hp.optional("Number of classes for the classification taks. (default: ``None``)", default=None)
```

Comment:
Wait why is this removed?

Reply:
This field is also on the base `ModelHparams` class, so it didn't need to be duplicated (eventually, we should remove it from the base class and only have it here.)